src/server.js

```diff
@@ -93,43 +93,48 @@ const apolloServer = createApolloServer(ApolloServer, {
 	context: createMicroContext,
 })
 
-const graphqlPath = '/api'
+const graphqlPath = `${ config.baseUrl }/api`
 const graphqlHandler = apolloServer.createHandler({ path: graphqlPath })
 
 const routes = [
 
-	get('/', async (req, res) => {
+	get(`${ config.baseUrl }/`, async (req, res) => {
 		res.setHeader('Content-Type', 'text/html; charset=utf-8')
 		res.end(await index)
 	}),
-	get('/index.html', async (req, res) => {
+	get(`${ config.baseUrl }/index.html`, async (req, res) => {
 		res.setHeader('Content-Type', 'text/html; charset=utf-8')
 		res.end(await index)
 	}),
-	get('/favicon.ico', async (req, res) => {
+	get(`${ config.baseUrl }/favicon.ico`, async (req, res) => {
 		res.setHeader('Content-Type', 'image/vnd.microsoft.icon')
 		res.end(await favicon)
 	}),
-	get('/index.css', async (req, res) => {
+	get(`${ config.baseUrl }/index.css`, async (req, res) => {
 		res.setHeader('Content-Type', 'text/css; charset=utf-8')
 		res.end(await styles)
 	}),
-	get('/index.js', async (req, res) => {
+	get(`${ config.baseUrl }/index.js`, async (req, res) => {
 		res.setHeader('Content-Type', 'text/javascript; charset=utf-8')
 		res.end(await scripts)
 	}),
-	get('/tracker.js', async (req, res) => {
+	get(`${ config.baseUrl }/tracker.js`, async (req, res) => {
 		res.setHeader('Content-Type', 'text/javascript; charset=utf-8')
 		res.end(await tracker)
 	}),
 	customTracker.exists === true ? get(customTracker.url, async (req, res) => {
```

Comment:
correct me if i'm wrong but i think you forgot adding baseUrl on customTracker.url and listening part
i do not know it gonn work but i think like this?
```javascript
customTracker.exists === true ?
  get(`${ config.baseUrl }${ customTracker.url }`, async (req, res) => {
    res.setHeader('Content-Type', 'text/javascript; charset=utf-8')
    res.end(await tracker)
  }) :
  undefined,
```

```javascript
server.on('listening', () => signale.watch(`Listening on http://localhost:${ config.port }${ config.baseUrl }`))
```

Reply:
Hey @sanguinary-bus, thx for your feedback!

I'll take a good look at this later. You might be right, I may have accidentally skipped the customtracker part.


> ``` js
> server.on('listening', () => signale.watch(`Listening on http://localhost:${ config.port }${ config.baseUrl }`))
> ```

I don't believe the above is necessary because my pull already changes the URL for each listening address in `/src/server.js`. If I recall correctly (>3 months ago...), I chose that path because of some oddity with ackee only wanting to listen to a path with trailing `/`. Even without `ACKEE_BASEURL` defined. That would break support for a very basic `https://ackee.mydomain.tld` without trailing `/`. I might take another look at that as well. 

But these are all moot points until @electerious is also able to commit his spare time to this pull.

Reply:
I'm happy to help, @dodedodo

the listening part maybe unnecessary, but a little bit detail around the logs definitely help :smile:

Reply:
Hey @sanguinary-bus, Sorry for the wait. 

Regarding the logs, you're completely right. I misinterpreted this as a namespace thing, which would be nice, but I don't think vercel micro supports it. 

Regarding the customTracker part, you're also completely right 😅. I forgot to test the `ACKEE_TRACKER` option. I've added the necessary changes.

@electerious could you take a look at this? It'd be much appreciated! :smile: